.go-version

```diff
@@ -1 +1 @@
-1.21.0
+1.23.0
```

Comment:
```suggestion
1.24.3
```

What do you think about updating it to the latest version? That would save us having to do it again later

Reply:
Good idea - I've updated to match the toolchain